core/src/test/java/com/linecorp/armeria/client/ClientMaxConnectionAgeTest.java

```diff
@@ -54,6 +58,14 @@ protected void configure(ServerBuilder sb) {
             sb.idleTimeoutMillis(0);
             sb.requestTimeoutMillis(0);
             sb.service("/", (ctx, req) -> HttpResponse.of(OK));
+            sb.annotatedService("/delayed", new Object() {
+
+                @Get
+                public HttpResponse delayed(@Param("seconds") String seconds) {
+                    return HttpResponse.delayed(
+                            HttpResponse.of(200), Duration.of(Long.valueOf(seconds), ChronoUnit.SECONDS));
```

Comment:
nit:
```suggestion
                            HttpResponse.of(200), Duration.ofSeconds(Long.valueOf(seconds)));
```
I think we can also change the type of `seconds` to `Long`(perhaps Duration) in the method definition.